pow/src/main/java/tech/pegasys/teku/pow/Eth1DepositManager.java

```diff
@@ -69,23 +70,55 @@ public void stop() {
     depositProcessingController.stopIfSubscribed();
   }
 
-  private SafeFuture<Void> headBeforeMinGenesisMode(EthBlock.Block headBlock) {
+  private SafeFuture<Void> processStart(
+      final EthBlock.Block headBlock, final ReplayDepositsResult replayDepositsResult) {
+    BigInteger startBlockNumber =
+        replayDepositsResult.getBlockNumber().orElse(UnsignedLong.ZERO).bigIntegerValue();
+    if (headBlock.getNumber().compareTo(startBlockNumber) > 0) {
```

Comment:
If the head block isn't after the start block number, we appear to not do anything and so never start tracking head. So even when the chain progresses past our start block we won't process any new blocks.

I suspect we need a `depositProcessingController.startSubscription(startBlockNumber)` in the else case for this outer if.

Reply:
good catch.

Reply:
So turns out I lied about the right solution.  If min genesis has already happened we need to run `depositProcessingController.startSubscription(startBlockNumber)` but if it hasn't we need to run `preGenesisSubscription(startBlockNumber)`.